packages/repl/prebake.mjs

```diff
@@ -36,6 +39,8 @@ export async function prebake() {
     samples(`${ds}/EmuSP12.json`),
     samples(`${ds}/vcsl.json`),
   ]);
+
+  aliasBank(`${ts}/tidal-drum-machines-alias.json`);
```

Comment:
what's the bet way to try out the repl? i only checked the web version works (with `pnpm dev`)

Reply:
i test with this https://github.com/tidalcycles/strudel/blob/main/examples/buildless/web-component-no-iframe.html replacing the unpkg url with the built repl package

Reply:
can u give an example of what the contents of your `web-component-no-iframe.html` file are when you do this? could write it up in the meantime too

Reply:
i've added infos on testing to `packages/repl/README.md` . also tested it and the alias stuff works :)